### PR TITLE
ADH-4387 [UI] Add Select, MultiSelect and depedent components

### PR DESCRIPTION
**Pull request by rimakan**
https://tracker.yandex.ru/ADH-4387

### CODE REVIEW

File: smart-frontend/app/src/components/uikit/Select/SingleSelect/SingleSelectList/SingleSelectList.module.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
+:global {
+  body.theme-dark {
+    --single-select-list-item-background: transparent;
+    --single-select-list-item-color: var(--text-default-color);
+
+    --single-select-list-item-background-hover: rgba(112, 122, 131, 0.2);
```
Comment (dbardin90) [inline smart-frontend/app/src/components/uikit/Select/SingleSelect/SingleSelectList/SingleSelectList.module.scss]: where is variable?

File: smart-frontend/app/src/styles/common.scss [hunk 2]
```diff
@@ -4,13 +4,17 @@ body {
     --text-default-color: var(--color-grayUsual);
     --text-green-color: var(--color-greenSaturated);
     --text-title-color: var(--color-offWhite);
+    --scrollbar-thumb-color: rgba(127, 130, 133, 0.20);
```
Comment (dbardin90) [inline smart-frontend/app/src/styles/common.scss]: no variable?

File: smart-frontend/app/src/styles/common.scss [hunk 3]
```diff
@@ -4,13 +4,17 @@ body {
...
+    --scrollbar-track-color: transparent;
   }
 
   &.theme-light {
     --body-background: var(--color-colourful);
     --text-default-color: var(--color-ADCM);
     --text-green-color: var(--color-greenLogo);
     --text-title-color: var(--color-dark1);
+    --scrollbar-thumb-color: rgba(127, 130, 133, 0.20);
```
Comment (dbardin90) [inline smart-frontend/app/src/styles/common.scss]: no variable?

File: smart-frontend/app/src/components/uikit/Select/SingleSelect/SingleSelectList/SingleSelectList.module.scss (hunk 1 above)
Comment (rimakan) [reply smart-frontend/app/src/components/uikit/Select/SingleSelect/SingleSelectList/SingleSelectList.module.scss]: fixed, thank you

File: smart-frontend/app/src/styles/common.scss (hunk 2 above)
Comment (rimakan) [reply smart-frontend/app/src/styles/common.scss]: fixed, thank you

File: smart-frontend/app/src/styles/common.scss (hunk 3 above)
Comment (rimakan) [reply smart-frontend/app/src/styles/common.scss]: fixed, thank you

**Review by remizov-arena — APPROVED** [review]

**Review by dbardin90 — APPROVED** [review]

**Pull request merged**